js/models/LocalSettings.js

```diff
@@ -29,11 +34,11 @@ export default class extends Model {
       errObj[fieldName].push(error);
     };
 
-    if (is.not.boolean(attrs.macStyleWinControls)) {
-      addError('macStyleWinControls', 'Please provide a boolean value.');
+    if (!controlStyles.includes( attrs.windowControlyStyle ) ) {
```

Comment:
I get you mean windowControlStyle -- yes there is an incorrect `y` after Control. Okay, cool with the lint rule.